app/views/static_pages/dashboard_steps/moderation_matrix.md

```diff
@@ -1,7 +1,14 @@
-### Moderation Matrix
-This matrix correlates with the [Community Rules](https://www.theodinproject.com/community_rules) that are required to be followed in order to participate in the community. Some rules have different levels of severity, so you may see references to them under different numbers of zaps.
```

Comment:
I might've missed it but we should also think about where we're going to link the additional guidelines at. I think placing them both at the end of the introduction section and at the end of the entire file would be the best. This way, someone who read the entire thing can go to them afterwards.

This is obviously moot if the additional guidelines are required reading during the new user onboarding process - then I'd put them at the top so whenever Matrix is referenced the guidelines are there as well.

Reply:
Good point. People will be required to read them both, but it wouldn't hurt to have a link on the matrix to the additional guidelines.